src/session.cpp

```
@@ -343,10 +350,8 @@ session_t::remote_endpoint() const {
         try {
             endpoint = ptr->socket->remote_endpoint();
         } catch(const std::system_error& e) {
-            // TODO: Log this.
+            // Ignore.
```

Are you sure, that to silently eat an exception is the best idea?